libs/libunibreak/DETAILS

```diff
@@ -1,12 +1,12 @@
           MODULE=libunibreak
-         VERSION=4_1
+         VERSION=5_0
           SOURCE=$MODULE\_$VERSION.tar.gz
-      SOURCE_URL=https://github.com/adah1972/libunibreak/archive/
+ SOURCE_URL_FULL=https://github.com/adah1972/libunibreak/releases/download/$MODULE_$VERSION/$MODULE-5.0.tar.gz
```

Comment:
Make it look like this;

SOURCE_URL=https://github.com/adah1972/libunibreak/releases/download/libunibreak_5_0/